api/v2010/call/create-example-10/output/create-example-10.json

```diff
@@ -26,7 +26,7 @@
     "feedback": "/2010-04-01/Accounts/ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX/Calls/CAXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX/Feedback.json",
     "feedback_summaries": "/2010-04-01/Accounts/ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX/Calls/FeedbackSummary.json"
   },
-  "to": "sip:kate@example.com?hatchkey=4815162342",
-  "to_formatted": "sip:kate@example.com?hatchkey=4815162342",
+  "to": "+14158675309",
+  "to_formatted": "(415) 867-5309",
```

Comment:
This doesn't look like what we want based on the example this is for:
https://github.com/twilio/sample-code/blob/master/api/v2010/call/create-example-10/create-example-10.3.x.js#L11